R/codebook.R

```diff
@@ -183,7 +183,7 @@ codebook <- function(df, path = NA, title = NA, subtitle = NA, description = NA)
 
     # Get column attributes
     table_var_attributes <- df %>%
-      codebook_get_col_attributes(col_nms[[i]]) %>%
+      codebook_get_col_attributes(colums[[i]]) %>%
```

Comment:
Are you sure about this change? I feel like `col_nms` (for column names) might be correct.

Reply:
You are right. I'm not sure why I changed it.

Reply:
Thank you!